.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,39 @@
+name: Automerge
```

Comment:
@cfernhout : I just thought of something.

This is a public repo. Someone can in theory raise a PR, then have a friend "approve" it maybe?  And then it automerges?  Or they can't because they're all read-only (not in the team)?

We should check that approval is only with us and not just anybody, and it cannot be abused that way.

Reply:
First-time commiters need to have approval. If we want we could also implement code owners such that at least a review from you or me is required.